src/Components/SmartComponents/SystemCves/SystemCveTableToolbar.js

```diff
@@ -94,7 +95,7 @@ const SystemCveToolbarWithContext = ({
             }}
             dedicatedAction={customAction ? (typeof customAction === 'function' ? customAction(cves) : customAction) : (entity &&
                 <Remediation
-                    isDisabled={selectedRowsRawData.length === 0}
+                    isDisabled={systemCVEs.payload.data?.length === 0}
```

Comment:
This should be disabled when there are no selected rows, not when there are no rows